src/app/shared/header/header.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Can we move the template location from app/shared/ to app/shared/components

Reply:
yes